sdk/python/kfp_tekton/compiler/_op_to_template.py

```diff
@@ -445,17 +432,6 @@ def _op_to_template(op: BaseOp, enable_artifacts=False):
     if processed_op.init_containers:
         template['spec']['steps'] = _prepend_steps(processed_op.init_containers, template['spec']['steps'])
 
-    # initial base_step and volume setup
-    base_step = {
-        'image': 'busybox',
-        'name': 'copy-results',
```

Comment:
I still see the `copy-results` step in the compiled YAML and in a comment, does that come from somewhere else?

Reply:
I moved this into a different function, but somehow it get put back after the merge. We no longer need this over here because it is initiated at the beginning of this function.

Reply:
right, thanks :-)